File: example/scalalib/testing/6-test-group-stealing/foo/test/src/GroupY5.scala
package foo
import utest._
object GroupY5 extends RandomTestsUtils {
  def tests = Tests {
    test("test1") { testGreeting("Xiuhtecuhtli", 26) }
    test("test2") { testGreeting("Yarilo", 22) }
    test("test3") { testGreeting("Zephyrus", 23) }
    test("test4") { testGreeting("Aegir", 25) }
  }
} 

File: example/kotlinlib/testing/6-test-group-stealing/foo/test/src/foo/GroupY5.kt
package foo

import org.junit.jupiter.api.Test

class GroupY5 : RandomTestsUtils() {
    @Test
    @Throws(Exception::class)
    fun test1() {
        testGreeting("Xiuhtecuhtli", 26)
    }

    @Test
    @Throws(Exception::class)
    fun test2() {
        testGreeting("Yarilo", 22)
    }

    @Test
    @Throws(Exception::class)
    fun test3() {
        testGreeting("Zephyrus", 23)
    }

    @Test
    @Throws(Exception::class)
    fun test4() {
        testGreeting("Aegir", 25)
    }
} 
